vts/coservsigner/icoservsigner.go

```diff
@@ -0,0 +1,17 @@
+// Copyright 2025 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+package coservsigner
+
+import (
+	"github.com/lestrrat-go/jwx/v2/jwa"
+	"github.com/lestrrat-go/jwx/v2/jwk"
+	"github.com/spf13/afero"
+	"github.com/veraison/corim/coserv"
+)
+
+type ICoservSigner interface {
+	Init(cfg Cfg, fs afero.Fs) error
```

Comment:
Should the `Init` be part of the interface? Surely, this will be specific to the implementation? (E.g. some other implementation may not rely on a file system, and won't take an `afero.Fs`).

Reply:
The reason why `afero.Fs` pops up here is because I am reusing the earsigner’s `KeyLoader`, which takes it to instantiate a new `KeyLoader` from which to load by URL.